man/rmd/template-makes-dummies.Rmd

```diff
@@ -0,0 +1 @@
+Factor/categorical predictors need to be converted to numeric values (e.g., dummy or indicator variables) for this engine. When using the formula method via \\code{\\link[=fit.model_spec]{fit.model_spec()}}, parsnip will convert factor columns to indicators.
```

Comment:
```suggestion
Factor/categorical predictors need to be converted to numeric values (e.g., dummy or indicator variables) for this engine. When using the formula method via \\code{\\link[=fit.model_spec]{fit()}}, parsnip will convert factor columns to indicators.
```

What if we just printed out `fit()` in situations like this? I've had people get confused about what they were supposed to type out, the generic or the whole method.